server/app/urls.py

```diff
@@ -0,0 +1,12 @@
+from django.urls import path, include
+from rest_framework.routers import DefaultRouter
+from .views import BidsViewSet
+
+# Create a router and register the bid viewset
+router = DefaultRouter()
+router.register(r'tasks/(?P<task_id>\d+)/bids', BidsViewSet, basename='bids')
```

Comment:
prefer to use django url dispatch instead of raw pattern. 

URL dispatcher: https://docs.djangoproject.com/en/5.0/topics/http/urls/#path-converters